DownloadAllFilesForOneProposalId.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+
+from pdart.astroquery.Astroquery import (
+    MastSlice,
+    CustomizedQueryMastSlice,
+    ProductSet,
+)
+
+from pdart.pipeline.Directories import DevDirectories
+
+# Download All files from Mast
+if __name__ == "__main__":
+    TWD = os.environ["TMP_WORKING_DIR"]
+    assert len(sys.argv) <= 3, sys.argv
+    proposal_id = int(sys.argv[1])
+    action = str(sys.argv[2]) if len(sys.argv) == 3 else None
+    path = f"{TWD}/files_from_mast"
+
+    dir = DevDirectories(path)
+    working_dir = dir.working_dir(proposal_id)
+    file_path = f"{working_dir}/{proposal_id:05}_files_info.txt"
+
+    if not os.path.isdir(working_dir):
+        os.makedirs(working_dir)
+    slice = CustomizedQueryMastSlice((1900, 1, 1), (2025, 1, 1), proposal_id)
+    result = slice.get_products(proposal_id)
+
+    # Get full list of files and unique suffixes of one proposal id
+    suffixes = []
+    with open(file_path, "w") as f:
+        f.write("==================\nList of files:\n==================\n")
+        for row in result:
+            productFilename = row["productFilename"]
+            productSubGroupDescription = row["productSubGroupDescription"]
+            # Using iterrows is faster but mypy will complain about Table dosen't
+            # have attribute iterrows. We can use # type: ignore to ignore it.
+            # for (productFilename, productSubGroupDescription) in result.iterrows(  # type: ignore
+            #     "productFilename", "productSubGroupDescription"
+            # ):
```

Comment:
`mypy` definitely can't do everything right now (and probably never will because the heads of Python don't like types).  So if you know that some of your code works but `mypy` complains, go ahead and tell it to shut up.  I've done that a few times in the codebase.